content-review/impl/urkund/src/main/java/org/sakaiproject/contentreview/urkund/UrkundSubmissionData.java

```diff
@@ -0,0 +1,34 @@
+package org.sakaiproject.contentreview.urkund;
+
+import java.util.Date;
+import java.util.Map;
+
+import lombok.Getter;
+import lombok.Setter;
+
+
+public class UrkundSubmissionData {
```

Comment:
"You can also put a @Getter and/or @Setter annotation on a class. In that case, it's as if you annotate all the non-static fields in that class with the annotation."